packages/keystone/test.utils.js

```diff
@@ -66,6 +68,37 @@ class UploadingFile {
     }
 }
 
+/**
+ * Returns correct file format based on environment
+ * @param {string} filePath - absolute path to the upload file on the local disk
+ * @param {Object} fileMeta - file meta, that should be attached to FileRecord
+ * @param {Object} user - result of 'createTestUser'
+ * @returns {Promise<UploadingFile>}
+ */
+async function getUploadingFile (filePath, fileMeta, user) {
```

Comment:
Looks like file will be `uploaded` in result
```suggestion
async function getUploadedFile (filePath, fileMeta, user) {
```
?